PC02_Graffiti_Imhoff.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Fri Sep  4 11:45:47 2020
+
+@author: hannahimhoff
+"""
```

Comment:
It's good practice to add a description to your code, so there is an idea of what your code is supposed to do!
```suggestion
This code draws various figures on an image of Jeff Bezos!
```